mlflow/utils/validation.py

```diff
@@ -51,7 +51,7 @@
 MAX_MODEL_REGISTRY_TAG_KEY_LENGTH = 250
 MAX_MODEL_REGISTRY_TAG_VALUE_LENGTH = 5000
 MAX_EXPERIMENTS_LISTED_PER_PAGE = 50000
-MAX_REGISTERED_MODEL_ALIAS_LENGTH = 256
+MAX_REGISTERED_MODEL_ALIAS_LENGTH = 255
```

Comment:
Is this a breaking change for OSS?

Reply:
Yeah marked it on the PR

Reply:
Got it, I'm ok with this given aliases just shipped, but we should double check with @dbczumar. It doesn't seem too terrible if the SqlAlchemyStore etc support one extra char in alias names (allows 256 char alias names) than UC does (only allows 255 char alias names) long term, if it actually comes up it's probably not too hard to support the extra char in UC

Reply:
Got it, I'm ok with this given aliases just shipped, but we should double check with @dbczumar. It doesn't seem too terrible if the SqlAlchemyStore etc support one extra char in alias names (allows 256 char alias names) than UC does (only allows 255 char alias names) long term, if it actually comes up it's probably not too hard to support the extra char in UC